.env

```diff
@@ -0,0 +1,3 @@
+APP_CONFIG_DEFAULT_SOURCE=/config.json
+APP_CONFIG_ALLOWED_SOURCES=gnnuzabk
+APP_UNSAFE_ALLOW_DYNAMIC_CONFIG=false
```

Comment:
Will we also use this value for the preview button on the App Config dashboard page?